pkg/external_repos/redhat_repos.json

```diff
@@ -142,5 +142,68 @@
         "arch": "x86_64",
         "distribution_version": "8",
         "feature_name": "OPENSHIFT-OCP-x86_64"
+    },
+    {
+        "name": "Red Hat Enterprise Linux 10 for x86_64 - BaseOS (RPMs)",
+        "content_label": "rhel-10-for-x86_64-baseos-rpms",
+        "url": "https://cdn.redhat.com/content/dist/rhel10/10/x86_64/baseos/os",
+        "arch": "x86_64",
+        "distribution_version": "10",
+        "selector": "rhel10",
+        "feature_name": "RHEL-OS-x86_64"
+    },
+    {
+        "name": "Red Hat Enterprise Linux 10 for x86_64 - AppStream (RPMs)",
+        "content_label": "rhel-10-for-x86_64-appstream-rpms",
+        "url": "https://cdn.redhat.com/content/dist/rhel10/10/x86_64/appstream/os",
+        "arch": "x86_64",
+        "distribution_version": "10",
+        "selector": "rhel10",
+        "feature_name": "RHEL-OS-x86_64"
+    },
+    {
+        "name": "Red Hat CodeReady Linux Builder for RHEL 10 x86_64 (RPMs)",
+        "content_label": "codeready-builder-for-rhel-10-x86_64-rpms",
+        "url": "https://cdn.redhat.com/content/dist/rhel10/10/x86_64/codeready-builder/os",
+        "arch": "x86_64",
+        "distribution_version": "10",
+        "selector": "rhel10",
+        "feature_name": "RHEL-OS-x86_64"
+    },
+    {
+        "name": "Red Hat Enterprise Linux 10 for ARM 64 - BaseOS (RPMs)",
+        "content_label": "rhel-10-for-aarch64-baseos-rpms",
+        "url": "https://cdn.redhat.com/content/dist/rhel10/10/aarch64/baseos/os",
+        "arch": "aarch64",
+        "distribution_version": "10",
+        "selector": "rhel10",
+        "feature_name": "RHEL-OS-x86_64"
+    },
+    {
+        "name": "Red Hat Enterprise Linux 10 for ARM 64 - AppStream (RPMs)",
+        "content_label": "rhel-10-for-aarch64-appstream-rpms",
+        "url": "https://cdn.redhat.com/content/dist/rhel10/10/aarch64/appstream/os",
+        "arch": "aarch64",
+        "distribution_version": "10",
+        "selector": "rhel10",
+        "feature_name": "RHEL-OS-x86_64"
+    },
+    {
+        "name": "Red Hat CodeReady Linux Builder for RHEL 10 ARM 64 (RPMs)",
+        "content_label": "codeready-builder-for-rhel-10-aarch64-rpms",
+        "url": "https://cdn.redhat.com/content/dist/rhel10/10/aarch64/codeready-builder/os",
+        "arch": "aarch64",
+        "distribution_version": "10",
+        "selector": "rhel10",
+        "feature_name": "RHEL-OS-x86_64"
+    },
+    {
+        "name":"Red Hat Enterprise Linux 10 for x86_64 - Extensions (RPMs)",
+        "content_label":"rhel-10-for-x86_64-extensions-rpms",
+        "url":"https://cdn.redhat.com/content/dist/rhel10/10/x86_64/extensions/os",
+        "arch":"x86_64",
+        "distribution_version":"10",
+        "selector": "rhel10",
+        "feature_name":"RHEL-OS-x86_64"
```

Comment:
This is actually available for arm too.  The features api seems broken for arm, i'll report that, but i think this should work:

```
{
        "name":"Red Hat Enterprise Linux 10 for ARM 64 - Extensions (RPMs)",
        "content_label":"rhel-10-for-aarch64-extensions-rpms",
        "url":"https://cdn.redhat.com/content/dist/rhel10/10/aarch64/extensions/os",
        "arch":"aarch64",
        "distribution_version":"10",
        "selector": "rhel10",
        "feature_name":"RHEL-OS-x86_64"
}

```

Reply:
added!